### DISCUSSION TITLE
Не отображается превью

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#383] [Как это сделать?] [answered]
Здравствуйте. Подскажите, пожалуйста, в чем может быть причина:
почему превью под Администратором показывает, а под другим Пользователем нет?
может я у роли Пользователя не включил какую-нибудь системную таблицу?
Спасибо

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/32614108/023f37ba-85fc-4892-8712-4f0a18af8461)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/32614108/7570fa87-7bd1-4b54-8efc-964c6a480c42)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Неизвестно — включите логи Селект или Форматирование и посмотрите

**Comment by tvideo** [comment]
Спасибо, через лог увидел ошибку.

**Reply by tvideo** [reply]
Получилось, что если в коде селлекта:
```
= : selectRowListForSelect(table: 'zayavki_all'; field: 'name_for_select'; bfield: 'base_field'; where: 'vyezd2' = #test; where: 'avtomobil' = ""; preview: 'avtomobil'; preview: 'voditel'; preview: 'vyruchka'; preview: 'zakazchik')
```
есть ссылка | where: 'vyezd2' = #test | на колонку, которая видна только Создателю, то превью не показывается у пользователя, у которого не видно это поле

**Reply by ttm-alexeypolunin** [reply]
Посмотрим подробнее, может так не должно быть те превью это иннер-канал
